opengrok-tools/src/main/python/opengrok_tools/sync.py

```diff
@@ -95,7 +95,7 @@ def main():
                         help='URI of the webapp with context path')
     args = parser.parse_args()
 
-    if args.debug:
+    git if args.debug:
```

Comment:
i guess windows has not been switched when you started typing ;-)

Reply:
d'oh